livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py

```diff
@@ -171,6 +171,9 @@ def __init__(
         )
         self._session = http_session
 
+        if _encoding_from_format(self._opts.encoding) == "mp3":
+            self._mp3_decoder = utils.codecs.Mp3StreamDecoder()
```

Comment:
Is initializing this class taking time?

Reply:
around  0.000267 seconds, previously it was around  0.24226 seconds

Reply:
right, so there is no need to move this code if the import is already done

Reply:
I've removed the code from tts classes